pkg/networkservice/chains/xconnectns/server_test.go

```diff
@@ -122,6 +123,7 @@ func tokenGenerator(peerAuthInfo credentials.AuthInfo) (tokenValue string, expir
 }
 
 func TestDirectMemifCase(t *testing.T) {
+	logrus.SetOutput(ioutil.Discard)
```

Comment:
I think We should return logging status, since it could lead to some comments missing for other tests.